readthedocs/config/config.py

```diff
@@ -670,6 +671,19 @@ class BuildConfigV2(BuildConfigBase):
         'dirhtml': 'sphinx_htmldir',
         'singlehtml': 'sphinx_singlehtml',
     }
+    valid_os = ['ubuntu-20.04']
```

Comment:
and, saw we were naming this as `ubuntu20`, any reason or objection into using a long form instead? `ubuntu-20.04`.

Reply:
This was decided like that just to simplify what the user has to write. We don't plan to support other versions that are not LTS, so `ubuntu20` covers all the `20.x` possible versions. There is not going to be `ubuntu-20.10`

Do you think it's better to explicitly define `ubuntu-20.04`? If that's the case, why?

Reply:
> Do you think it's better to explicitly define ubuntu-20.04? If that's the case, why?

It's more explicit for the user what to expect, if I see ubuntu-20, feels like it's the latest release from 20.x, not the latest LTS.

Reply:
I don't have a strong opinion on this, I guess.

I think it's easier to make typos and generate failing builds with those extra chars (in particular, `-` and `.`). Besides, the expectation about what version will be used will be clarified with the documentation: "Read the Docs only supports Ubuntu LTS versions: `ubuntu20` will use the Ubuntu 20.04 LTS version"

Reply:
I'm 👍  on the more explicit name. Gives us more flexibility in the future if we need it.